app/controllers/appeal-application/confirmation-page.ts

```diff
@@ -24,6 +25,8 @@ function getConfirmationPage(req: Request, res: Response, next: NextFunction) {
 }
 
 function getConfirmationPayLaterPage(req: Request, res: Response, next: NextFunction) {
+  req.app.locals.logger.trace(`Successful pay later submission ${JSON.stringify(req.session.appeal.ccdCaseId)}`, 'Confirmation appeal submission');
```

Comment:
@gavinslilley if you can update to

`Successful AIP pay later submission for ccd id`